apps/resources/forms.py

```diff
@@ -78,3 +78,9 @@ def save(self, commit=True):
         if self.cleaned_data['is_public']:
             self.cleaned_data['privacy'] = ''
         return super().save(commit=commit)
+
+    def clean_title(self):
+        title = self.cleaned_data['title']
+        if Resource.objects.filter(slug=slugify(title)).exists():
+            raise forms.ValidationError('Resource with this title already exist')
```

Comment:
exist -> exists

Reply:
Maybe add an error code and a test - though it's not super important.

Reply:
Yes, but a unique code so that when you test it you're confident that this is the exact exception that gets raised during testing - and not some other ValidationError elsewhere in code.